GettingStarted.md

```diff
@@ -51,5 +51,7 @@ You will probably want to add `ManualCameraControl.cs` (found under
 
 Deploy to the emulator or device.
```

Comment:
This steps describe how to deploy from the HoloToolkit/Build Window - which doesn't expose platform and doesn't require specifying it. Maybe update the note to refer to the particular case of deploying from within Visual Studio?